pkg/cleaner/cleaning.go

```diff
@@ -88,6 +90,7 @@ func (c *FileProcessor) Process(path string) error {
 }
 
 func (c *FileContentObfuscator) ObfuscateFile(inputFile string, outputFile string) error {
+	var fileExt int
```

Comment:
why define that at the top? you only use it in the error scope below

Reply:
The scope of "fileExt" would be alive in case of calling this method by the same caller from multiple threads.
Thus, I felt, the number of checks, to verify if a file exists, would reduce.
Ex: 
Caller-1 called this method from Go-routine-1, Let's say fileExt has become 3
Caller-2 called this method from Go-routine-2, it can start checking the existence of file from fileExt = 4 i.e "xxx-xxx-xxx-xxx.yaml.4"

Anyway, the scope can also be limited to the error scenario only.

Reply:
each goroutine has their own stack, so for each goroutine this variable will be 0 from the start anyway and they will each individually increment until they find a non-conflicting extension.